federation/backends/stac_catalogs.md

```diff
@@ -0,0 +1,80 @@
+# STAC Catalog profile
+
+This document lists the requirements for a data provider that wants to make a dataset available as a 
+new collection in the openEO platform. More specifically, it defines the requirements for STAC catalogs. The STAC specification
+is also used within openEO, and is a common choice in the EO community.
+
+The requirements and guidelines in this document serve mostly as clarifications to STAC itself. It does not aim to contradict or overrule
+the STAC specification, and if this is perceived to be the case, the STAC specification should get precedence. It may however further constrain
+STAC in cases where it allows too much freedom.
+
+## Scope
+
+This profile aims to add a minimal set of requirements to STAC that is needed for a catalog to be compatible with openEO backends.
+Requirements that are optional, for instance because they enhance efficiency, are indicated as such.
+
+This profile is limited in scope to the most common types of raster EO datasets in the sense of how they are organized. It does not aim to define guidelines
+for very complex cases, or cover evey possible type of (EO) dataset. 
+
+### Dataset organization
+
+1. STAC collections will be mapped to openEO collections
+2. All STAC products in the same collection have the same set of 'bands', that are defined at collection level. This allows backends to look up the selected bands in a product.
+3. Assets in a product are raster files that contain a single band. 
```

Comment:
This is not clear to me. What is meant by product? A single item? If that is stored as GTIFF it can contain also several bands.